### PR TITLE
MMS 2402: Adding placer and contacts to NOW ETLs

### PR DESCRIPTION
**Pull request by pdittaro**
This PR adds on to the existing MMS NoW ETLs to transform placer and contact information.

### CODE REVIEW

**Comment by JustinMacaulay** [comment]
This PR should be good for review now. I had to do some weird work arounds to get placer to insert correctly but after looking at the data and looking in the MMS tables I am not sure it is storing the placer information the same way. I would appreciate any help validating the data and any feedback on the script.

File: pipeline/config-dev.groovy [hunk 1]
```diff
@@ -253,7 +253,7 @@ app {
 environments {
     'dev' {
         vars {
-            DB_PVC_SIZE = '1Gi'
+            DB_PVC_SIZE = '10Gi'
```
Comment (pdittaro) [inline pipeline/config-dev.groovy]: Remember to resize these environment variables to the appropriate sizes when you are done testing.

Comment (JustinMacaulay) [reply pipeline/config-dev.groovy]: Yeah I was gonna revert them when I knew the PR was approved

File: microservices/now_etls/NOW_import.py [hunk 2]
```diff
@@ -94,7 +92,7 @@ def NOW_submissions_ETL(connection):
         # Importing the vFCBC NoW submission data.
         print('Beginning vFCBC NoW ETL:')
         ETL_MMS_NOW_schema(connection, SHARED_TABLES, 'mms_now_vfcbc', 'VFCBC')
-        # connection.commit()
+        connection.commit()
```
[the remark this answers is not in the document]
Comment (JustinMacaulay) [reply microservices/now_etls/NOW_import.py]: Removed this commit statement

**Pull request merged**